pkg/controller/migmigration/migrate.go

```diff
@@ -83,7 +70,7 @@ func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) (int, er
 		Client:          r,
 		Owner:           migration,
 		PlanResources:   planResources,
-		Phase:           Phase{Name: migration.Status.Phase},
+		Phase:           migration.Status.Phase,
```

Comment:
Another possible merge issue, or are does this commit change the way we specify phase here?

Reply:
This is intentional.  Since phases are not repeated, the ordered conditions is no longer useful and this PR intentionally reverts that functionality to reduce complexity.